doc/context.md

```diff
@@ -0,0 +1,290 @@
+# Axe Testing Context
+
+Axe-core's `context` argument is a powerful tool for controlling precisely which elements are tested and which are ignored. The context lets you do many things, including:
+
+1. [Test specific elements](#test-specific-elements)
+1. [Test DOM nodes](#test-dom-nodes)
+1. [Excludes elements from test](#exclude-elements-from-test)
+1. [Select from prior tests](#select-from-prior-tests)
+1. [Limit frame testing](#limit-frame-testing)
+1. [Limit shadow DOM testing](#limit-shadow-dom-testing)
+1. [Combine shadow DOM and frame context](#combine-shadow-dom-and-frame-context)
+1. [Implicit frame and shadow DOM selection](#implicit-frame-and-shadow-dom-selection)
+
+## Test Specific Elements
+
+When passed a CSS selector or array of CSS selectors, axe will test only the elements that match those selectors, along with any content inside those elements:
+
+```js
+// Test every <nav> and <main> element, and everything inside them:
+await axe.run('nav, main');
+// Test every <nav> element, every element with the sideBar class,
+// and the element with the #header ID, along with all their content:
+await axe.run(['nav', '.sideBar', '#header']);
+```
+
+**Tip**: Sometimes pages are worked on by multiple teams. If you mark your team's sections up with a unique class, axe can use that class to only test the sections your team works on.
+
+Axe-core is commonly used through browser driver APIs such as Selenium, Puppeteer, and Playwright. Most of these APIs do not pass a context object in directly. Instead, the following is achieved through the `.include()` method. Visit the docs of your API for more information. For most cases, this will look like the following:
+
+```js
+const axe = new AxeBuilder({ page });
+// Test every <nav> and <main> element, and everything inside them:
+axe.include('nav, main');
+```
+
+## Test DOM Nodes
+
+Axe can accept native DOM elements for testing. These must be attached to the DOM tree first. Testing DOM elements is useful in test environments such as Karma and Jest. The following example shows axe testing DOM elements:
+
+```js
+// Test a single DOM node
+const img = document.createElement('img');
+document.body.appendChild(img);
+await axe.run(img);
+
+// Test a Node list:
+const nodes = document.querySelector('main, header');
+await axe.run(nodes);
+
+// Test an array of nodes:
+const navbar = document.getElementById('navbar');
+const cookiePopup = document.getElementById('cookie-popup');
+await axe.run([navbar, cookiePopup]);
+```
+
+### Component Frameworks
```

Comment:
If we're listing component frameworks, could we also list a small section for testing frameworks like Jest? Then we could add a quick example showing that you need to mount instead of shallowMount a component for testing

Reply:
Good point, but I'm going for a note instead.